advanced/keyboard-shortcuts.md

```diff
@@ -63,4 +63,15 @@ This is a list of available keyboard shortcuts within the editor user interface.
 
 ## Add custom shortcuts to TinyMCE
 
-Refer to the [addShortcut]({{site.baseurl}}/api/tinymce/tinymce.editor/#addshortcut) and [tinymce.Shortcuts]({{site.baseurl}}/api/tinymce/tinymce.shortcuts/) sections to add custom keyboard shortcuts to {{site.productname}}.
+> **Important**: Adding a custom shortcut that with a keyboard combination that conflicts with an existing shortcut, the existing shortcut will be overridden.
```

Comment:
`that with a keyboard` probably shouldn't have `that` here?

Reply:
ok, that sentence is nonsense. updating.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/d6f9cf4a8078de019a983938369aca2dfef00e0b